### DISCUSSION TITLE
[Feature Request]: 联动同步删除后，再下载该剧集能否不要自动删除？

### DISCUSSION THREAD

**Discussion by NikoYomi** [jxxghp/MoviePilot#3381] [讨论]
### 当前程序版本

v1.3.9-2

### 运行环境

Docker

### 功能改进类型

插件

### 功能改进

开启了插件【媒体文件同步删除】，同步方式为-日志。
当我在媒体服务器上删除一个文件时，他能够同步删除所有的源文件和链接文件。这很好。但是当我再下载该剧集时候，他居然在转移入库之后，自动识别之前的日志给我删除了。

期望能够在删除该剧集后，再次下次入库时，不要再自动删除了。

ps：
我想插件运行是否没有识别日志发生的时间？
比如当我下载《钢铁侠》时，发现他的画面质量不符合预期而删除。
此时插件识别到并删除的日期是2023.11.06 08:10。将此时间记录下来。
那么当我重新下载和入库的时间肯定是在之后的，插件就可以避免删除了。

**Comment by dlovew** [comment]
最简单的就是同步方式改为插件或者webhook

**Comment by NikoYomi** [comment]
> 最简单的就是同步方式改为插件或者webhook

我设置了webhook和scripter，但是不知道为什么不起作用。当然这是我自己的问题。也没找到网上相关的教程。

**Comment by thsrite** [comment]
日志方式更新插件1.2版本试试

**Comment by LoSiTeE** [comment]
> 日志方式更新插件1.2版本试试

不行，还是会删除

**Comment by LoSiTeE** [comment]
> > 最简单的就是同步方式改为插件或者webhook
> 
> 我设置了webhook和scripter，但是不知道为什么不起作用。当然这是我自己的问题。也没找到网上相关的教程。

scripter貌似只能用在emby上，而jellyfin自带的webhook插件没有item delete这个选项，也就是说条目删除的时候jellyfin是不会发送信息给MP的，所以无解
